webpack/webpack.config.babel.js

```diff
@@ -22,7 +22,7 @@ module.exports = {
         },
       },
       {
-        test: /\.scss$/,
+        test: /\.(sc|c)ss$/,
```

Comment:
Kind of that?

```/\.s?css$/```

Reply:
Although, you might have different stack for SCSS and pure CSS files. For SASS/SCSS you will have preprocessor, and probably postprocessor. For pure/enriched CSS you will need only PostCSS. Basically, future CSS cannot be processed by SASS so it's better to split.

Not this time.